packages/slate/src/interfaces/element.js

```diff
@@ -1763,13 +1763,12 @@ class ElementInterface {
     const newParentPath = PathUtils.lift(newPath)
     this.assertNode(newParentPath)
 
-    const [p, np] = PathUtils.crop(path, newPath)
-    const position = PathUtils.compare(p, np)
```

Comment:
Minor issue, but we don't really need `p` and `np` since `compare` only checks the smaller size anyway. This avoids the two calls to `.slice` that `crop` makes. We just use `path.size` in the conditional below because we know it is smaller than `newPath` in order to hit that line